packages/ua-devtools-evm-hardhat/package.json

```diff
@@ -55,6 +55,7 @@
     "@layerzerolabs/protocol-devtools-evm": "~1.2.1",
     "@layerzerolabs/ua-devtools": "~1.0.5",
     "@layerzerolabs/ua-devtools-evm": "~3.0.1",
+    "@layerzerolabs/ua-devtools-solana": "~2.0.0",
```

Comment:
This breaks the architecture and results in purely EVM projects to download all the Solana stuff